docs/codeql/codeql-cli/publishing-and-using-codeql-packs.rst

```diff
@@ -72,3 +72,39 @@ The ``analyze`` command will run the default suite of any specified CodeQL packs
 ::
 
    codeql <database> analyze <scope>/<pack> <scope>/<other-pack>
+
+Managing packs on GitHub Enterprise Server
+------------------------------------------
+
```

Comment:
For extra visibility, it might be worth adding an extra `.. pull-quote:` here calling out that this is only available for the container registry on GHES 3.6 and later.